src/main/java/uk/gov/justice/probation/courtcaseservice/service/OffenderMatchService.java

```diff
@@ -87,6 +88,20 @@ public OffenderMatchDetailResponse getOffenderMatchDetailsByCaseIdAndDefendantId
         return OffenderMatchDetailResponse.builder().offenderMatchDetails(offenderMatchDetails).build();
     }
 
+    public List<OffenderMatchDetail> getOffenderMatchDetailsByDefendantId(String defendantId) {
```

Comment:
Service returning controller DTO

Reply:
This service method, apart from fetching the entities, makes rest calls to offender search API for each offender match fetched from the database enriches the returned entities.. so it is not the entities that are returned to the client but the response from the offender search API that is returned, hence the DTOs